tests/run-build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+export RKT_STAGE1_USR_FROM=$1
+export RKT_STAGE1_SYSTEMD_VER=$2
+
+export BUILD_DIR=build-rkt-$RKT_STAGE1_USR_FROM-$RKT_STAGE1_SYSTEMD_VER
+
+mkdir -p builds
+cd builds
+
+# Semaphore does not clean git subtrees between each build.
+rm -rf $BUILD_DIR
+
+git clone ../ $BUILD_DIR
```

Comment:
Good idea. The disk space on Semaphore is limited, so let's reduce the risks of getting a full disk.